src/Maestro/Client/src/Generated/Channels.cs

```diff
@@ -619,32 +619,6 @@ internal async Task OnRemoveBuildFromChannelFailed(Request req, Response res)
             CancellationToken cancellationToken = default
         )
         {
-
```

Comment:
Why are these lines being deleted?

Reply:
They make it impossible to call the API. Some of the parameters are of type `bool` and this code throws when the value is `false` so you have no way to pass `false` to API. Similar for `channelId` - `0` is valid number accepted by API but this code throws when it sees it. Probably this could be solved by configuring generator properly but I'm not familiar with that.

Reply:
I see. Makes sense.

Reply:
These are generated files, and so will get overwritten as soon as someone regenerates the client here. @alexperovich knows most about how the client generator works (I believe it lives in arcade). I'm okay with removing these temporarily, but we need to fix the generator. Can you file an issue?

Also, it looks like the default checks for default(int) can stay.

Reply:
Yea, the generator needs to be fixed. This should be a simple fix, I'll explore it.

Reply:
https://github.com/dotnet/arcade/blob/master/src/Microsoft.DotNet.SwaggerGenerator/Microsoft.DotNet.SwaggerGenerator.CodeGenerator/Languages/csharp/MethodGroup.hb#L123-L128

I am VERY surprised we haven't hit this before...do we have any other REST apis that take bools?

Reply:
I think they are all optional bools, so the parameter becomes `bool?`

Reply:
Yeah I think these could be optional.

Reply:
Okay, I fixed the generator checking for default(int) and other value types, but its still broken because some of the parameters have no type.

Reply:
Somehow the enumeration parameters have been broken, and they don't show up as `"type": "string", "enum": [ ... ]` in the swagger json. The generator defaults them to "void" because they have no defined type, and the code doesn't compile.

Reply:
crap, its just something not supported in swagger v2, we configured swashbuckle to use referenced definitions for enums, so there wasn't a new type for each usage of the enum, but references aren't supported for enums in that location in the swagger for version 2. I guess we just have to switch back to the old mode.

Reply:
Okay, I got it fixed. I'll send the pr for the minor changes to the swagger generation now, and then send the generator fix.

Reply:
@alexperovich I updated the generator in Version.Details.xml and Version.props and rebuilt all the models. I had to temporarily switch swagger URL in csproj to localhost because API changes do not exist on staging yet. Hopefully everything's fine but I would appreciate it if you could take a quick look.